src/psyclone/configuration.py

```diff
@@ -449,9 +449,18 @@ def find_file():
         if not within_virtual_env():
             # 4. <python-installation-base>/share/psyclone/
             _file_paths.append(share_dir)
+
         # 5. <psyclone-installation-base>/share/psyclone/
         _file_paths.extend(pkg_share_dir)
 
+        # 6. <psyclone-src-base>/config/
+        # Search for configuration file relative to this source file
+        dev_path_list = os.path.split(
+                            os.path.abspath(__file__))[:-1]+(
+                                "..", "..", "config")
+        dev_path = os.path.abspath(os.path.join(*dev_path_list))
+        _file_paths.append(dev_path)
```

Comment:
This was more intended to solve the problems of editable mode rather than getting into how to set up tests based on psyclone installed in editable mode.
Did you had something easier in mind?

Reply:
Granted, but you've modified the source so we need a test that ensures that e.g. `_file_paths` is now set as expected. Hopefully, this simply means extending `tests/config_test.py::test_search_path`.

Reply:
Now I understand what you mean.
I found all the checks and added also a check (including a check for the lowest priority for this search path).